app/helpers/samples_helper.rb

```diff
@@ -28,7 +28,7 @@ def generate_sample_list_csv(formatted_samples)
                         location: db_sample ? db_sample[:sample_location] : '',
                         host_genome: derived_output ? derived_output[:host_genome_name] : '',
                         notes: db_sample ? db_sample[:sample_notes] : '',
-                        overall_job_status: run_info ? run_info[:job_status_description] : '',
+                        overall_job_status: run_info ? run_info[:result_status_description] : '',
                         host_filtering_status: run_info ? run_info['Host Filtering'] : '',
```

Comment:
is it planned that these (host_filtering_status, nonhost_alignment_status, postprocessing_status) will also go away from here eventually?

Reply:
sure